.registryrc

```diff
@@ -1 +1 @@
-e3330d51ebad77e2aa62c34d5a74746ae5e83e64
+73f635d983dfce5e4a4979f0af1d717147cf68f8
```

Comment:
update once https://github.com/hyperlane-xyz/hyperlane-registry/pull/279 is merged